yapapi/rest/payment.py

```diff
@@ -130,12 +130,12 @@ def new_allocation(
         """Creates new allocation.
 
         - `amount`:  Allocation amount.
-        - `expires`: expiration timestamp. by default 30 minutes from now.
+        - `expires`: expiration timestamp. by default 10 years.
```

Comment:
please create an issue describing the underlying problem and link to the issue in a "@todo" below

```suggestion
        - `expires`: expiration timestamp. set to default 10 years as a work-around.
```